man/existsBackgroundVariables.Rd

```diff
@@ -1,38 +1,31 @@
 \name{existsBackgroundVariables}
 \alias{existsBackgroundVariables}
-%- Also NEED an '\alias' for EACH other topic documented here.
 \title{Internally needed function for consistency checks and data preparation.}
 \description{Function is necessary for \code{eatRep} and \code{eatModel} as well
 and therefore exported to namespace.}
 \usage{
-existsBackgroundVariables (dat, variable, warnIfMissing = FALSE)
+existsBackgroundVariables (dat, variable, warnIfMissing = FALSE,
+     stopIfMissingOnVars = NULL)
 }
-%- maybe also 'usage' for other objects documented here.
 \arguments{
   \item{dat}{
-%%     ~~Describe \code{file} here~~
 A data frame
 }
   \item{variable}{
-%%     ~~Describe \code{file} here~~
 column number or variable name
 }
   \item{warnIfMissing}{
-%%     ~~Describe \code{file} here~~
 Logical: gives a warning if the variable contains missing values
+}
+  \item{stopIfMissingOnVars}{
+Optional: Applies only if \code{warnIfMissing = TRUE}. Character vectors with
+variable names where missings are strictly forbidden. If missings occur on these
```

Comment:
Nit: Second sentence is a bit confusing (and I am not 100% sure how it works). Maybe:
Warnings as raised through \code{warnIfMissing = TRUE} are turned into errors..

Reply:
Yes, good idea ... thanks!